src/explore-education-statistics-admin/src/index.tsx

```diff
@@ -1,13 +1,16 @@
 import '@admin/polyfill';
 import configureAxios from '@admin/services/utils/configureAxios';
+import { createRoot } from 'react-dom/client';
 import React from 'react';
-import ReactDOM from 'react-dom';
 import * as serviceWorker from './serviceWorker';
 
 configureAxios();
 
 import('./App').then(({ default: App }) => {
-  ReactDOM.render(<App />, document.getElementById('root'));
+  const container = document.getElementById('root');
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
Purely a suggestion, not a blocking comment - but since we're adding a line here anyway to disable the es-lint rule, I wonder if a "if not container throw error" line would be preferable, for the sake of failing fast?

Reply:
Good idea, I've done that now.